ui/src/app/utils/access-utils.tsx

```diff
@@ -185,18 +192,23 @@ export const getAccessModuleConfig = (
         requiredForRTAccess: enforceRasLoginGovLinking,
         requiredForCTAccess: enforceRasLoginGovLinking,
 
-        DARTitleComponent: () => (
-          <div>
-            Verify your identity with Login.gov{' '}
-            <TooltipTrigger
-              content={
-                'For additional security, we require you to verify your identity by uploading a photo of your ID.'
-              }
-            >
-              <InfoIcon style={{ margin: '0 0.3rem' }} />
-            </TooltipTrigger>
-          </div>
-        ),
+        DARTitleComponent: (props: DARTitleComponentConfig) => {
+          return !!props.profile ? (
```

Comment:
I do not love this line. RASLINKLOGINGOV should always require a profile in order for LoginGovHelpText to work correctly. However, if this line is removed, the application seems to function correctly, but there is a single test that fails in data-access-requirements.spec.tsx on line 406:

```
  // Temporary hack Sep 16: when enableRasLoginGovLinking is false, we DO show the module
  // along with an Ineligible icon and some "technical difficulties" text
  // and it never becomes an activeModule
  it('should render the RAS module as ineligible when its feature flag is disabled', () => {
```